Cannot build in Windows 10 using build instructions

### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running a recent nightly build or the *newest* stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Following the Windows build instructions while installing both Visual Studio and Qt Creator for the fist time, build fails in both with the following errors:

Visual Studio - 

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		CMake Error at C:\Users\Johnny\Source\Repos\chatterino2\CMakeLists.txt:43 (find_package):
  By not providing "FindQt5.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5" with any of
  the following names:

    Qt5Config.cmake
    qt5-config.cmake

  Add the installation prefix of "Qt5" to CMAKE_PREFIX_PATH or set "Qt5_DIR"
  to a directory containing one of the above files.  If "Qt5" provides a
  separate development package or SDK, be sure it has been installed.	chatterino	C:\Users\Johnny\Source\Repos\chatterino2\CMakeLists.txt	43	

```
  
Qt Creator - 

```
C:\local\boost\boost\asio\ssl\detail\openssl_types.hpp:23: error: C1083: Cannot open include file: 'openssl/conf.h': No such file or directory
```

Also, instructions are outdated.  OpenSSL 1.0 is no longer available for download, and the download links provided lead to an error page.  I installed OpenSSL 1.1.1m instead.

### Screenshots

_No response_

### OS and Chatterino Version

Windows 10 64, d21bdc6

--- a/BUILDING_ON_WINDOWS.md
+++ b/BUILDING_ON_WINDOWS.md
@@ -33,7 +33,7 @@ Note: This installation will take about 1.5 GB of disk space.
 
 ### For our websocket library, we need OpenSSL 1.1
 
-1. Download OpenSSL for windows, version `1.1.1l`: **[Download](https://slproweb.com/download/Win64OpenSSL-1_1_1L.exe)**
+1. Download OpenSSL for windows, version `1.1.1m`: **[Download](https://slproweb.com/download/Win64OpenSSL-1_1_1m.exe)**
 2. When prompted, install OpenSSL to `C:\local\openssl`
 3. When prompted, copy the OpenSSL DLLs to "The OpenSSL binaries (/bin) directory".
 

